src/index.js

```diff
@@ -101,16 +130,11 @@ class ServerlessOfflineAwsEventbridgePlugin {
     });
 
     this.app.all("*", async (req, res) => {
-      const invokedLambdas = this.invokeSubscribers(req.body.Entries);
-      if (invokedLambdas.length) {
-        const eventResults = await Promise.all(invokedLambdas);
-        res.json({
```

Comment:
Returning the right response is something that was added in a previous PR. This is the same response that AWS returns. Leaving this out may cause issues for some users